sources/packages/web/src/components/aest/students/bypassedRestrictions/HistoryBypassedRestrictions.vue

```diff
@@ -107,6 +147,44 @@ export default defineComponent({
       return !isBypassActive || !isRestrictionActive ? "secondary" : "primary";
     };
 
+    const openBypassRestrictionModal = async () => {
+      const result = await bypassRestrictionModal.value.showModal({
+        applicationId: props.applicationId,
+      });
+      if (result) {
+        reloadRestrictionBypassedHistory();
+      }
+    };
+    const openRemoveBypassModal = async (
+      applicationRestrictionBypassId: number,
+    ) => {
+      const result = await removeBypassModal.value.showModal({
+        applicationRestrictionBypassId,
+      });
+      if (result) {
+        reloadRestrictionBypassedHistory();
+      }
+    };
+
+    const openBypassViewDetailsModal = async (
+      applicationRestrictionBypassId: number,
+    ) => {
+      await bypassRestrictionModal.value.showModal({
+        applicationRestrictionBypassId,
+      });
+    };
+
+    const loadBypassedRestrictionHistory = async () => {
+      bypassedRestrictions.value =
+        await ApplicationRestrictionBypassService.shared.getApplicationRestrictionBypassesHistory(
+          props.applicationId,
+        );
+    };
+
+    const reloadRestrictionBypassedHistory = () => {
+      loadBypassedRestrictionHistory();
+    };
```

Comment:
Is this method required? It seems that is is just "renaming" the method `loadBypassedRestrictionHistory` to `reloadRestrictionBypassedHistory`.